APE14.rst

```diff
@@ -0,0 +1,484 @@
+data. While we use pixel as a shorthand, this need not be only two-dimensional
+(the typical usage of *pixel*) - it includes one-dimensional (e.g., spectral
+axes), or more-than-2-dimensional (e.g., data cubes). Similarly *world* in this
+context is a generalized concept of any sort of physical coordinate used in the
+data. Exactly what that physical interpretation is is not specified in the
+concept of WCS. For the purposes of this APE, *world* could even be an
+intermediate state of knowledge. E.g., for the unity transformation the world
+coordinate *is* the pixel coordinate, for example if no calibration information
+has yet been determined. But some sense of direction and a language for the
+physical type and representation of a world coordinate needs to be conveyed in
+any practical implementation of WCS. A common understanding of how to convey
+that information, and represent it in Python, is the primary subject of this
+APE.
+
+Scope
+^^^^^
+
+This APE is not meant to limit further development on more generalized
+coordinate transform APIs. A future APE may define the API for accessing
+“intermediate frames” as used in gWCS or the LSST transform systems. This APE
+is only intended to define a clear API for converting from pixel coordinates to
+a specific end set of “world” coordinates (as defined above). This APE doesn’t
+focus on how to get intermediate frames if they are present, or combining
+together multiple steps of a transformation - while such functionality is useful
+and should be *compatible* with this APE’s interface, an API for these
+operations is out-of-scope for this APE. This APE also does not address how to
+*create* such WCSs. Rather it is aimed specifically at an API for the simpler
+problem of actually performing the “end-to-end” transformation from pixel to
+some world space.
+
+Overview of the proposed WCS Interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Our proposal in this APE is the following:
+
+* To develop a common API that WCS objects can expose, either directly or
+  through thin wrappers, and which will allow functionality accessing the WCS
+  objects to be agnostic of the actual WCS representation. We want this API to
+  be as simple as possible and mainly return simple Python objects such as
+  strings, lists and arrays. We refer to this as the ‘low-level’ API.
+
+* To develop a high-level Astropy WCS object that can wrap any WCS object
+  providing the low-level API and be smarter about the kinds of objects to
+  accept or return to the user (including for example SkyCoord or Time objects).
+
+The idea of this two-tiered approach as opposed to a single-tier approach is
+that if we asked different WCS objects to provide a high-level API, this would
+(1) cause a lot of duplication of logic of constructing the appropriate Astropy
+objects, and (2) force those objects to return specifically Astropy objects,
+whereas having a package-independent API would be better.
```

Comment:
Can we add something like "better because it allows other packages, even non-python packages, to follow a well-defined API for the sake of interoperability."

Or something else saying specifically why it would be better?  My suggestion is a mouthfull...